en_US.json

```diff
@@ -1639,6 +1639,7 @@
 	"propertyListing.advancedFilter.invalid.value.message": "Invalid value",
 	"propertyListing.advancedFilter.keyword.title": "Keywords",
 	"propertyListing.advancedFilter.tooltip.button": "Advanced Filter",
+	"propertyListing.attack.migration.tooltip": "attack mitigation enabled",
```

Comment:
I think the first character should be uppercase i.e. "Attack mitigation enabled" instead of "attack mitigation enabled".

You can confirm with Burton / Bin.

Reply:
@chiraags  good catch.  Yes, we usually capitalize the first word.
Also, the string name says "migration" when it ought to be "mitigation".
These aren't the same things.

Reply:
Thanks for reviewing.
I changed it to uppercase and id of the string.